solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceProducerProperties.java

```diff
@@ -1,13 +1,27 @@
 package com.solace.spring.cloud.stream.binder.properties;
 
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+@ConfigurationProperties("spring.cloud.stream.solace.default.producer")
```

Comment:
See review comment for `SolaceConsumerProperties`.

Reply:
Done